test/helper.go

```diff
@@ -28,6 +30,16 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 )
 
+func HasDocker() bool {
+	if runtime.GOOS != "linux" {
```

Comment:
I am assuming this is also true for WSL/Windows :)

Reply:
I had to google 'WSL', Windows Subsystem for Linux.. yeah not worried about that :)